jdaviz/configs/imviz/plugins/conesearch/conesearch.py

```diff
@@ -0,0 +1,30 @@
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astroquery.sdss import SDSS
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+
+__all__ = ['ConeSearch']
+
+
+@tray_registry('imviz-conesearch', label="Imviz ConeSearch")
+class ConeSearch(PluginTemplateMixin, ViewerSelectMixin):
+    template_file = __file__, "conesearch.vue"
+
+    def __init__(self, *args, **kwargs):
+        self.init_astrowidgets_api()
+        super().__init__(*args, **kwargs)
+
+    def vue_do_conesearch(self, *args, **kwargs):
+        query_region_result = SDSS.query_region(SkyCoord('00h07m13.8036s +14d56m16.8706s', frame='icrs'), radius='0.015 deg', data_release=17)
```

Comment:
You would want the `SkyCoord` and `radius` be calculated from the displayed field of view (i.e., what you can see in Imviz display). For this purpose, I would point you here:

https://github.com/spacetelescope/jdaviz/blob/cf22f84dc6eb75d21db6907dab9894433e1e9db4/jdaviz/configs/imviz/plugins/viewers.py#L248

You would only need the `pixel_to_world` part because you want the sky coordinates and radius. You might need to play around with the code to see if you need special logic to handle use case when non-reference image is displayed.

`SkyCoord` takes more than string inputs, so also look at https://docs.astropy.org/en/stable/coordinates/index.html

Reply:
Would it make more sense to convert the output of `_get_zoom_limits(self, image)` back to world coordinates, or just re-use portions of the code from that function in my own file?

Reply:
I used the function Pey-Lian provided as inspiration to obtain the center point of the displayed field of view.

Reply:
Email said you asked a question but I don't see it now, so maybe you figured it out? I did write a "get zoom center" method over at #1384 for something else but I am not sure if it is necessary. You can have a look if you want.

`jdaviz/configs/imviz/plugins/viewers.py`

```python
    def _get_zoom_center(self, image):
        """Return ``(x, y)`` of the center of zoom for a given image.
        Also see :meth:`_get_zoom_limits`.
        """
        x_cen = (self.state.x_min + self.state.x_max) * 0.5
        y_cen = (self.state.y_min + self.state.y_max) * 0.5
        return self._get_real_xy(image, x_cen, y_cen)[:2]
```

Reply:
It was a question about code re-use, but then it didn't end up applying to my solution, sorry about that! The code you provided just now is exactly what I did to get the center, should I just add that to `viewers.py`?

Reply:
Sure, I can see how it can be useful for other stuff down the road. Thanks!